composer.json

```diff
@@ -1,34 +1,43 @@
 {
-    "name": "aws/aws-sdk-php-zf2",
-    "description": "Zend Framework 2 Module that allows easy integration the AWS SDK for PHP",
-    "type": "library",
-    "license": "Apache",
-    "keywords": ["aws", "zf2", "amazon", "zend", "s3", "dynamodb", "ec2", "cloudfront", "sqs"],
-    "require": {
-        "php": ">=5.5",
-        "aws/aws-sdk-php": "3.*",
-        "zendframework/zend-filter": "2.*",
-        "zendframework/zend-servicemanager": "2.7.* || 3.*",
-        "zendframework/zend-session": "2.*",
-        "zendframework/zend-version": "2.*",
-        "zendframework/zend-view": "2.*"
+  "name": "firelike/aws-sdk-php-zf2",
```

Comment:
don't change the name, to use your own repo for test, you can define "repositories" key in your composer's json to define your fork url.